connector/src/main/java/org/geysermc/connector/dump/DumpInfo.java

```diff
@@ -64,9 +68,13 @@
     private final Object2IntMap<DeviceOs> userPlatforms;
     private final HashInfo hashInfo;
     private final RamInfo ramInfo;
+    private LogsInfo logsInfo;
     private final BootstrapDumpInfo bootstrapInfo;
 
-    public DumpInfo() {
+    public DumpInfo(boolean addLog) {
+        if (addLog) {
+            this.logsInfo = new LogsInfo(addLog);
+        }
```

Comment:
could this be moved down between 
```java
this.ramInfo = new DumpInfo.RamInfo();

this.userPlatforms = new Object2IntOpenHashMap<>();
```
just so that order of DumpInfo() lines up with the order of the fields

Reply:
Oh also, if you check addLog before creating LogsInfo, you don't need to pass addLog to LogsInfo

Reply:
😂